src/main/java/pl/tomaszdziurko/jvm_bloggers/metadata/MetadataKeys.java

```diff
@@ -0,0 +1,10 @@
+package pl.tomaszdziurko.jvm_bloggers.metadata;
+
+public enum MetadataKeys {
```

Comment:
Same for me

@tdziurko Please decide whether we stay with convesion of enums being containers for constants or this can be recaftored.